pkg/gui/controllers/local_commits_controller.go

```diff
@@ -947,6 +995,10 @@ func (self *LocalCommitsController) createAmendCommit(commit *models.Commit, inc
 						return err
 					}
 
+					if err := self.moveFixupCommitToCurrentBranchOfStack(commit); err != nil {
```

Comment:
Likewise I'd call this `moveFixupCommitToOwnerStackedBranch`. Using `Current` just makes me think about the current branch. Thoughts?

Reply:
Changed to that in 9553e304b35e. Again, I wonder if `moveFixupCommitToTargetBranch` is better.

Reply:
I don't like target branch because the word 'target' seems arbitrary to me; like using 'selected' branch: it implies that something is doing the targeting and that it could have targeted something else. But a commit isn't really targeting the branch: it's owned by the branch.

I agree that 'owner' is still awkward but I think it's clearer than 'target'.